### DISCUSSION TITLE
Revision graph

### DISCUSSION THREAD

**Discussion by WeiZhou-dev** [Satish-Mutyala/T24S2T1EXTG12#7] [General]
Hi, I am checking the revision graph and noticed the branch names all start with the origin. Please let me know if this matters or if we need to remove it.
![scrennshot](https://github.com/user-attachments/assets/5dc04733-acde-44dc-ab45-c60c89b2d276)

**Comment by JeongseopYun** [comment]
Hello Wei, in Git, "origin" is a shorthand name for the remote repository that a project was originally cloned from.

**Reply by Satish-Mutyala** [reply]
Hi @WeiZhou-dev It was the name you given when you setup the remote. That is perfectly all right.
